docs/DesignDocs/AttestationQPLEnforcementSupport.md

```diff
@@ -0,0 +1,70 @@
+Attestation SGX QPL Enforcement Support
+====
+
+This proposal is to make a change to `oe_verify_evidence`, so that it can support policy-based SGX QPL (Quote Provider Library) specific enhancement, such as allowing the OE SDK applications to pass third party cloud specific parameters to the SGX QPL for more advanced quote verification.
```

Comment:
```suggestion
This proposal is to make a change to `oe_verify_evidence`, so that it can support policy-based SGX QPL (Quote Provider Library) specific enhancement, such as allowing Verifier applications to pass third party cloud specific parameters to the SGX QPL to allow them to do more advanced quote verification.
```